src/plugins/intel_gna/tests/unit/gna_memory_alignment.cpp

```diff
@@ -149,16 +149,14 @@ INSTANTIATE_TEST_SUITE_P(MemoryAlignment_GNA_4_0,
 
 class MemoryAlignmentTest : public ::testing::Test {};
 
-TEST(MemoryAlignmentTest, getMemoryAlignmentBytes_ExpectExceptionWhenTargetIsUnset) {
-    EXPECT_ANY_THROW(getMemoryAlignmentBytes(DeviceVersion::NotSet));
-}
-
 TEST(MemoryAlignmentTest, getMemoryAlignmentBytes_Expect64ByteAlignmentWhenTargetIsGNA3_0) {
-    EXPECT_EQ(getMemoryAlignmentBytes(DeviceVersion::GNA3_0), 64);
+    Limitations::GetInstance().Init(DeviceVersion::GNA3_5);
```

Comment:
If you intended to change GNA3_0 to GNA3_5, please also update the test case name

Reply:
ok